cli/src/commands/init/index.ts

```diff
@@ -2,19 +2,32 @@ import { Flags } from '@oclif/core';
 import { getCurrentBranchName } from '@xata.io/client';
 import chalk from 'chalk';
 import { spawn } from 'child_process';
+import { highlight } from 'cli-highlight';
 import { access, readFile, writeFile } from 'fs/promises';
 import path from 'path';
 import which from 'which';
 import { createAPIKeyThroughWebUI } from '../../auth-server.js';
 import { BaseCommand } from '../../base.js';
 import { getProfile } from '../../credentials.js';
+import { isIgnored } from '../../git.js';
 import { xataDatabaseSchema } from '../../schema.js';
+import Browse from '../browse/index.js';
 import Codegen from '../codegen/index.js';
+import RandomData from '../random-data/index.js';
+import EditSchema from '../schema/edit.js';
+import Shell from '../shell/index.js';
 
 export default class Init extends BaseCommand {
   static description = 'Configure your working directory to work with a Xata database';
 
-  static examples = [];
+  static examples = [
+    'Initialize a new project',
+    'xata init --db https://workspace-1234.xata.sh/db/database-name',
+    'Initialize a new project using a schema dump',
+    'xata init --db https://workspace-1234.xata.sh/db/database-name --schema schema.json',
+    'Initialize a new project without flags. The workspace and database will be asked interactively',
+    'xata init'
```

Comment:
Yep. Because if you don't specify a db, the command requires a global api key. With the db flag it doesn't require the user to have any key previously configured